scripts/release-helm.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -euo pipefail
+
+helm_charts_installation_id() {
+	JWT=$1
+	curl -s -X GET -H "Accept: application/vnd.github+json" \
+		-H "Authorization: Bearer ${JWT}" \
+		-H "X-GitHub-Api-Version: 2022-11-28" \
+		"https://api.github.com/repos/mongodb/helm-charts/installation" | jq .id
+}
+
+helm_charts_token() {
+	JWT=$1
+	INSTALL_ID=$(helm_charts_installation_id "${JWT}")
+	curl -s -X POST -H "Accept: application/vnd.github+json" \
+		-H "Authorization: Bearer ${JWT}" \
+		-H "X-GitHub-Api-Version: 2022-11-28" \
+		"https://api.github.com/app/installations/${INSTALL_ID}/access_tokens" | jq -rc .token
+}
+
+JWT=$(tools/makejwt/makejwt -appId="${APP_ID}" -key="${RSA_PEM_KEY_BASE64}")
```

Comment:
Can JWTbe scriptized too? make sense do it?

Reply:
You mean turn **makejwt** into a script?
I would say no in our particular case. The examples we had for make JWT were in Python and Ruby, meaning that they are too much for bash script readability. In our case we did not have Ruby or Python dependencies here, so I would rather just use go for the tricky code tooling, just as we did with the clean tool.